packages/block-editor/src/store/selectors.js

```diff
@@ -2647,7 +2647,7 @@ export const __experimentalGetActiveBlockIdByBlockNames = createSelector(
 export function wasBlockJustInserted( state, clientId, source ) {
 	const { lastBlockInserted } = state;
 	return (
-		lastBlockInserted.clientId === clientId &&
+		lastBlockInserted.clientIds?.includes( clientId ) &&
```

Comment:
The idea here is to preserve the intent of the existing selector as if the block has just been inserted then it will stil be a valid result.